### ISSUE TITLE
[BUG] News, changelog, and copyright tabs don't work

### ISSUE THREAD

**Issue by JakeHasSkill** [PojavLauncherTeam/PojavLauncher#3103]
### Describe the bug

On the news tab, the Changelog, News, and Copyright buttons do not show anything when tapped on even though they appear highlighted/selected after tapped.

### The log file and images/videos

![Screenshot_20220418-172800-960](https://user-images.githubusercontent.com/68882341/163892648-39df35ec-dcfd-413f-b3b5-708ff8e8a9e9.png)

### Steps To Reproduce

```markdown
- Open pojav launcher and sign in
- Try tapping any of the tabs on the news page
```

### Expected Behavior

I should be able to read the news which might have useful information

### Platform

```markdown
- Device model: moto g stylus
- CPU architecture: aarch64 (Qualcomm SDM665 Snapdragon 665)
- Android version: 11
- PojavLauncher version: crocus-v3_openjdk
```

### Anything else?

I tried scrolling down too.

**Comment by kuzy21** [comment]
This also happens to me, I think it affects the newer updates of pojavlauncher.

**Comment by ajit56** [comment]
@artdeell

**Comment by mchnjt** [comment]
> @artdeell

~~why~~

**Comment by mchnjt** [comment]
It should be fixed now after #3062 gets merged
